src/Illuminate/Database/Query/Builder.php

```diff
@@ -1402,18 +1404,20 @@ public function paginate($perPage = 15, $columns = ['*'])
 	 *
 	 * This is more efficient on larger data-sets, etc.
 	 *
-	 * @param  int  $perPage
-	 * @param  array  $columns
+	 * @param  int $perPage
+	 * @param  array $columns
```

Comment:
you get the point...

Reply:
@GrahamCampbell that is the most arrogant condescending review I have ever seen!  
Grow up please